src/praisonai/praisonai/llm/env.py

```
@@ -39,6 +39,68 @@ class LLMEndpoint:
+        if not os.environ.get(key_var):
+            continue
+        if catalogue is not None:
+            # Validate the bare model id (catalogue ids are unprefixed) but
+            # keep the provider prefix so endpoint routing still works. If the
+            # catalogue can't confirm it, use the model as-is rather than
+            # dropping a perfectly valid provider default.
+            bare = model.split("/", 1)[-1]
+            try:
+                catalogue.validate_model(bare)
+            except Exception:
+                pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)